packages/sit-onyx/src/components/OnyxRipple/types.ts

```diff
@@ -0,0 +1,6 @@
+export type OnyxRippleProps = {
+  enterDuration?: string;
+  leaveDuration?: string;
```

Comment:
I think for now its fine to define a fixed duration so its uniform across all components :)

We have some pre-defined durations from UX, maybe we can use `var(--onyx-duration-sm)`?
![image](https://github.com/user-attachments/assets/b5984f26-e811-4ec2-8cdb-6a50447b1c90)

Reply:
changed it, duration-sm is fine for enter, for leave it's way too long, I kept 100ms